packages/frontend/src/components/common/PivotTable/index.tsx

```diff
@@ -266,10 +325,104 @@ const PivotTable: FC<PivotTableProps> = ({
                                     />
                                 );
                             })}
+
+                            {/* render the total values */}
+                            {data.pivotConfig.rowTotals
+                                ? data.rowTotals?.[rowIndex].map(
+                                      (total, colIndex) => {
+                                          const value = getRowTotalItemFromAxis(
+                                              total,
+                                              rowIndex,
+                                              colIndex,
+                                          );
+
+                                          return total ? (
+                                              <TotalCell
+                                                  value={value}
+                                                  key={`index-total-${rowIndex}-${colIndex}`}
+                                              >
+                                                  {value.formatted}
+                                              </TotalCell>
+                                          ) : (
+                                              <td
+                                                  className={cellCx(
+                                                      cellStyles.root,
+                                                  )}
+                                              />
+                                          );
+                                      },
+                                  )
+                                : null}
                         </>
                     </tr>
                 ))}
             </tbody>
+
+            {/* TODO: column totals */}
+            {false && data.pivotConfig.columnTotals ? (
+                <tfoot>
+                    {data.columnTotals?.map((row, totalRowIndex) => (
+                        <tr key={`column-total-${totalRowIndex}`}>
+                            {/* shows empty cell if row numbers are visible */}
+                            {hideRowNumbers ? null : (
+                                <th
+                                    className={cellCx(
+                                        cellStyles.root,
+                                        cellStyles.rowNumber,
+                                    )}
+                                />
+                            )}
+
+                            {/* render the total label */}
+                            {data.columnTotalHeaders?.[totalRowIndex].map(
+                                (totalLabel, totalColIndex) =>
+                                    totalLabel ? (
+                                        <HeaderCell
+                                            key={`footer-total-${totalRowIndex}-${totalColIndex}`}
+                                            textAlign={
+                                                totalLabel.titleDirection ===
```

Comment:
Why do we change alignment for total cells?

Reply:
in the footer yes, but removed direction from types as it is clear now that totals on the footer will be aligned right and row totals will be default left.